### PR TITLE
Update readme + remove unused files

### PR DESCRIPTION
**Pull request by YannickdeHoop**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,180 +1,26 @@
...
+This stack provides a ROS driver for the SICK lidar sensors mentioned in the following list.
 
 ## Supported Hardware
 
 This driver should work with all of the following products.
 
-ROS Device Driver for SICK lidar and radar sensors - supported scanner types:
-
-
+ROS Device Driver for SICK lidar and radar sensors - supported scanner types:\
```
Comment (reinzor) [inline README.md]: remove `\`

File: README.md [hunk 2]
```diff
@@ -1,180 +1,26 @@
...
-The hostname is the ip-address of the scanner:
-
-```bash
-roslaunch <launch-file> hostname:=<ip-address>
-```
-e.g.
+- For TiM771S:
 ```bash
-roslaunch sick_scan sick_tim_5xx.launch hostname:=192.168.0.71
+rosrun sick_scan sick_generic_caller __name:=sick_tim_7xxS _hostname:=10.39.46.23
```
Comment (reinzor) [inline README.md]: what about the `scanner_type: sick_tim_7xxS` ?

File: README.md [hunk 3]
```diff
@@ -1,180 +1,26 @@
...
-
-### Start Multiple Nodes
-
-Take the launchfile "sick_tim_5xx_twin.launch" as an example.
-Rempping the scan and cloud topics is essential to distinguish the scanndata and provide TF information.
-
-## Parameter
-
-The use of the parameters can be looked up in the launch files. This is also recommended as a starting point.
 ### Common parameters
```
Comment (reinzor) [inline README.md:24]: Parameters do not align with:

```
hostname: 10.39.46.23
frame_id: rear_scan
scanner_type: sick_tim_7xxS
min_ang: -1.5
max_ang: 1.5
port: 2112
intensity: false
auto_reboot: false
use_binary_protocol: true
timelimit: 1.0
time_offset: -0.06
```

**Review by reinzor — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (YannickdeHoop) [reply README.md:21]: that's what the driver expects

**Review by reinzor — APPROVED** [review]

**Pull request merged**